action/protocol/staking/handler_candidate_selfstake_test.go

```diff
@@ -84,7 +84,11 @@ func initTestStateWithHeight(t *testing.T, ctrl *gomock.Controller, bucketCfgs [
 	}, &BuilderConfig{
 		Staking:                  genesis.Default.Staking,
 		PersistStakingPatchBlock: math.MaxUint64,
-	}, nil, nil, nil, genesis.Default.GreenlandBlockHeight)
+		Revise: ReviseConfig{
+			VoteWeight:         genesis.Default.Staking.VoteWeightCalConsts,
+			CorrectCandsHeight: genesis.Default.GreenlandBlockHeight,
```

Comment:
in our mainnet/default config should be:
CorrectCandsHeight: OkhotskBlockHeight
ReviseHeights: []{GreenlandBlockHeight, HawaiiBlockHeight}

Reply:
could just leave the heights empty since the test does not use reviser code